README.md

```diff
@@ -1,6 +1,160 @@
-# Anthos Service Mesh packages
+# ASM MCP
+
+This repository contains a blueprint repository for installing the ASM MCP
```

Comment:
Add the full name for MCP? Istio previously has a Mesh Config Protocol also with abbrev MCP, so this may be misleading.